api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -210,11 +208,8 @@ public ParticipantCohortAnnotation apply(org.pmiops.workbench.db.model.Participa
 
     Cohort cohort = cohortReviewService.findCohort(cohortId);
     //this validates that the user is in the proper workspace
-    Workspace workspace = cohortReviewService.validateMatchingWorkspace(workspaceNamespace,
-      workspaceId, cohort.getWorkspaceId(), WorkspaceAccessLevel.WRITER);
-
-    CdrVersionContext.setCdrVersion(workspace.getCdrVersion());
-
+    cohortReviewService.validateMatchingWorkspace(workspaceNamespace,
```

Comment:
I find setting the CDR version to be a surprising side-effect of this method name. Perhaps rename or else do not embed that logic there.

Reply:
Renamed.